rdrf/rdrf/features/steps.py

```diff
@@ -581,3 +699,61 @@ def clear_file_upload(step, section, cde, download_name):
         raise Exception("Could not click the file clear checkbox")
     
 
+@step('when I scroll to section "(.*)"')
+def scroll_to_section(step, section):
+    from selenium.webdriver.common.action_chains import ActionChains
+    mover = ActionChains(world.browser)
+    print("scrolling to section %s" % section)
+    section_xpath = ".//div[@class='panel panel-default' and contains(.,'%s') and not(contains(., '__prefix__')) and not(contains(.,'View previous values'))]" % section
+    section_element = world.browser.find_element_by_xpath(section_xpath)
+    if not section_element:
+        raise Exception("could not find section %s" % section)
+    y = scroll_to(section_element)
+    mover.move_to_element(section_element)
+    print("scrolled to section %s y = %s" % (section, y))
+
+
+@step('I click the add button for multisection "(.*)"')
+def add_multisection_item(step, section):
+  xpath = ".//div[@class='panel-heading' and contains(.,'%s') and not(contains(., '__prefix__')) and not(contains(.,'View previous values'))]" % section
+  div = world.browser.find_element_by_xpath(xpath)
+  add_link_xpath = """.//a[starts-with(@onclick,"add_form('formset_")]"""
+  add_link = div.find_element_by_xpath(add_link_xpath)
+  add_link.click()
+  # sometimes the next cde send keys was going to the wrong item
+  wait_n_seconds(step, 5)
```

Comment:
Not sure why this explicit sleep is needed, but in general I always ask if they can be replaced with waiting for something to appear etc. with a timeout.